local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -26,6 +26,12 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 )
 
+const (
+	// kubelet event reason list
+	FailedCleaningLocalPV = "FailedCleaningLocalPV"
```

Comment:
I didn't see any existing events that seemed quite right, but FailedUnmount or FailedDetachVolume might be close enough. 

Here are the four Volume related kevents:

```
FailedAttachVolume                   = "FailedAttachVolume"
FailedDetachVolume                   = "FailedDetachVolume"
FailedMountVolume                    = "FailedMount"
FailedUnMountVolume                  = "FailedUnMount"

```

https://github.com/kubernetes/kubernetes/blob/886e04f1fffbb04faf8a9f9ee141143b2684ae68/pkg/kubelet/events/event.go#L50

@msau42 - what do you prefer?